app/uk/gov/hmrc/taxenrolmentassignmentfrontend/views/LandingPage.scala.html

```diff
@@ -0,0 +1,38 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.Layout
+
+@this(
+        layout: Layout,
+        govukButton: GovukButton
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = Some(messages("landingPage.title"))) {
+
+ <h2 class="govuk-heading-xl">@messages("landingPage.heading")</h2>
+ <p class="govuk-body">@messages("landingPage.paragraph")</p>
+
+ <div class="govuk-button-group">
```

Comment:
This div is not need?

Reply:
Don't see it in the design system either so will remove.